CryptKeeperZK/crypt-keeper-extension
# refactor: one identity list

## Explanation

This PR is for removing the linked and the unlinked tabs, and just having one single identities list. 
 
## Related Issues

* Blocked by #875 
* Fixes #943 

## Screenshots

![Screenshot_20230928_203406](https://github.com/CryptKeeperZK/crypt-keeper-extension/assets/36774944/7f404d72-1d54-4bc8-a013-af4c7622f1a4)


## Manual Testing Steps

- Go to the demo
- Check the connect identity list

## Pre-Merge Checklist


> PR template source from [github.com/MetaMask](https://github.com/MetaMask)
